help/en/html/tools/logixng/reference/chapter2.shtml

```diff
@@ -27,6 +27,28 @@
       <p>It also possible to <strong>copy</strong> all or part of a ConditionalNG. This can then be
       pasted into the same ConditionalNG, another ConditionalNG or even a different LogixNG.</p>
 
+      <p>The LogixNG context menu contains four items for working with the clipboard.</p>
+
+      <div style="margin-left: 2em">
+        <a href="images/chapter2/clipboard_actions.png"><img class="image-border" src=
+        "images/chapter2/clipboard_actions.png" alt="Chapter 2 clipboard actions"></a>
+      </div>
+
+      <dl>
+        <dt>Cut</dt>
+        <dd>The row and its children are removed from the ConditionalNG and added the clipboard.</dd>
+
+        <dt>Copy</dt>
+        <dd>The row and its children are copied from the ConditionalNG and added the clipboard.</dd>
+
+        <dt>Paste</dt>
+        <dd>The first row in the clipboard is pasted to the empty row and removed from the clipboard.</dd>
+
+        <dt>Paste copy</dt>
+        <dd>The first row in the clipboard is pasted to the empty row but it is not removed from the
```

Comment:
A copy of the first row in the clipboard is pasted...

Normally the difference doesn't matter, but if the user is looking at the system names and/or the user names, he will notice. Since actions and expressions are NamedBean's, they must have valid system names and user names that are unique.

So this option makes a deep copy of the entire sub tree before pasting it.